doc/v2/documentation_data_format.rst

```diff
@@ -14,187 +15,375 @@ This document explains the PEtab data format.
+  assignment rather than time derivatives. They are not necessarily constant,
+  for example, the targets of SBML assignment rules.
 
-This is specified as a tab-separated value file in the following way:
+* **Constant entities**: Entities are that are defined in terms of a constant
+  value but may be subject to event assignments, e.g., parameters of an SBML
+  model that are not targets of rate rules or assignment rules.
 
-+--------------+------------------+------------------------------------+-----+---------------------------------------+
-| conditionId  | [conditionName]  | modelEntityId1                     | ... | modelEntityId${n}                     |
-+==============+==================+====================================+=====+=======================================+
-| STRING       | [STRING]         | NUMERIC\|STRING                    | ... | NUMERIC\|STRING                       |
-+--------------+------------------+------------------------------------+-----+---------------------------------------+
-| e.g.         |                  |                                    |     |                                       |
-+--------------+------------------+------------------------------------+-----+---------------------------------------+
-| conditionId1 | [conditionName1] | 0.42                               | ... | parameterId                           |
-+--------------+------------------+------------------------------------+-----+---------------------------------------+
-| conditionId2 | ...              | ...                                | ... | ...                                   |
-+--------------+------------------+------------------------------------+-----+---------------------------------------+
-| ...          | ...              | ...                                | ... | ..                                    |
-+--------------+------------------+------------------------------------+-----+---------------------------------------+
+.. _v2_conditions_table:
 
-Row- and column-ordering are arbitrary, although specifying ``conditionId``
-first may improve human readability.
+Conditions table
+----------------
 
-Additional columns are *not* allowed.
+The conditions table defines discrete changes to the model. These (sets of)
+changes typically represent interventions, perturbations, or changes in the
+environment of the system of interest. These modifications are referred to as
+(experimental) *conditions*.
+
+Conditions are applied at specific time points, which are defined in the
+:ref:`v2_experiments_table`. This allows for the specification of time
+courses or experiments spanning multiple time periods. A time period is the
+interval between two consecutive time points in the experiments table for a
+given experiment or the time between the last time point of an experiment and
+the time point of the last measurement for that experiment in the
+:ref:`v2_measurements_table`.
```

Comment:
Since this is a user's choice, I don't think we need to specify that the simulation stops at the last measurement. A tool might want to give users an easier way to perform predictions, than providing a dummy measurements table.

Reply:
Change to "between the last time point and the end of the simulation" ?

Reply:
Updated.

Reply:
I could imagine adding an empty time interval at the end without any condition with the time>last timepoint. I.e. one could have a time interval at the end where no measurements exist. In this case the simulation has to run until the last specified time point of the experiment.
This should be clarified and would provide a simple manner to encode the end of the simulation if one wants, e.g. for visualization or other purposes.

Reply:
I'd consider that out of scope here. For the purpose of parameter estimation, tools can just stop at the last measurement timepoint.

Your use case would have to be addressed in some visualization function. For proper visualization, additional information such as the output timepoints would be required. I don't think we should try to get that in here.

Reply:
Or could I just add a dummy timepoint to the measurement table with a NaN value at the end?

Reply:
> Or could I just add a dummy timepoint to the measurement table with a NaN value at the end?

That would currently be considered an error. NaN is not allowed. We had that implemented at some point, but it's an abuse of the measurements table. We can continue that discussion outside this PR if you'd consider it important to change that.

If the only purpose is visualization, I'd rather specify the endpoint and potentially time-resolution in the visualization table (or any better replacement of that...). 

Let's get this thing here merged and revisit any revision of visualization-related functionality afterwards?